scripts/build/check_init_priorities_test.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+"""
+Tests for check_init_priorities
+"""
+
+import mock
+import os
+import sys
+import unittest
+
+from elftools.elf.relocation import RelocationSection
+from elftools.elf.sections import SymbolTableSection
+
+ZEPHYR_BASE = os.getenv("ZEPHYR_BASE")
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/build"))
```

Comment:
it is not required to have ZEPHYR_BASE in environment, so please use `Path(__file__).resolve()` or `os.path.dirname(__file__)`.

Reply:
...is this even needed?! Seems to work without it, let me try and drop it entirely.

Reply:
Works, go figure... gone.